mobile/lib/common/settings/user_screen.dart

```diff
@@ -61,6 +65,154 @@ class _UserSettingsState extends State<UserSettings> {
+                                    ))
+                              ])),
+                          Text("$numberOfActivatedReferrals/$numberOfTotalReferrals",
+                              style: const TextStyle(fontSize: 18)),
+                        ],
+                      ),
+                      const Divider(),
+                      Visibility(
+                          visible: bonusStatusType == rust.BonusStatusType.Referral,
+                          child: Column(
+                            children: [
+                              Row(
+                                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                                children: [
+                                  const Text("Referral Level", style: TextStyle(fontSize: 18)),
+                                  Text(referralTierName, style: const TextStyle(fontSize: 18)),
+                                ],
+                              ),
+                              Row(
+                                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                                children: [
+                                  const Text("Referral Tier Level", style: TextStyle(fontSize: 18)),
+                                  Text("$referralTier", style: const TextStyle(fontSize: 18)),
+                                ],
+                              ),
+                            ],
+                          )),
+                      Visibility(
+                          visible: bonusStatusType == rust.BonusStatusType.Referent,
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                            children: [
+                              const Text("Referral Tier", style: TextStyle(fontSize: 18)),
+                              Text(referralTierName, style: const TextStyle(fontSize: 18)),
+                            ],
+                          )),
+                      const SizedBox(height: 5),
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          const Text("Active referral bonus", style: TextStyle(fontSize: 18)),
+                          Text("$referralFeeBonus%", style: const TextStyle(fontSize: 18)),
+                        ],
+                      ),
+                      const SizedBox(height: 10),
+                      const Row(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: [
+                          Text('*',
+                              style: TextStyle(
+                                fontFeatures: <FontFeature>[
+                                  FontFeature.superscripts(),
+                                ],
+                              )),
+                          Text("Some referents have not traded enough yet.")
```

Comment:
We could probably phrase that more positively by adding the asterix next to the activated users. --> referred user that have traded.